src/pages/levels/Controls.js

```diff
@@ -0,0 +1,113 @@
+import { useNavigate } from '@reach/router';
+import {
+  FormControl,
+  FormControlLabel,
+  Grid,
+  InputLabel,
+  MenuItem,
+  Select,
+  Switch,
+} from '@material-ui/core';
+import React from 'react';
+import Popularity from 'components/Popularity';
+import styled from 'styled-components';
+
+const Controls = ({ detailed, sort }) => {
+  const navigate = useNavigate();
+
+  return (
+    <Grid2
+      container
+      justify="flex-end"
+      alignItems="center"
+      className="controls"
+    >
+      {!detailed && (
+        <div style={{ paddingBottom: 4 }}>
```

Comment:
no inline styling plz

Reply:
i feel like its simpler this way but ok i'll take a look. don't get me wrong i do like styled components but I also like the simpler approach to things and I just don't feel like they are always the better choice. It wasn't out of laziness that i did this, but it was intentional, because it feels better to me. Using a styled component adds 3 more lines of code and extra little indirection when reading the component because you don't know what the styled component does until you look to the bottom. Yes that is the point of styled components but for one-off styles I just don't see how much that really benefits. Anyways.. like i said i'll take a look.